test/unit/base.test.js

```diff
@@ -292,6 +293,50 @@ describe('CONVERT FUNCTION TESTS ', function() {
         done();
       });
     });
+    it('convertor should maintain implicit headers in the request' +
+    implicitHeadersSpec, function(done) {
+      var openapi = fs.readFileSync(implicitHeadersSpec, 'utf8');
+      Converter.convert({ type: 'string', data: openapi }, {
+        schemaFaker: true,
+        keepImplicitHeaders: true
+      }, (err, conversionResult) => {
+        expect(err).to.be.null;
+        expect(conversionResult.output[0].data.item[0].item[0].request.header[0].key)
```

Comment:
Add following test to make sure there are only 2 headers generated.

```
expect(conversionResult.output[0].data.item[0].item[0].request.header).to.have.length(2);
```

Reply:
I corrected the test as suggested